src/backwardproblem.cpp

```diff
@@ -178,36 +127,19 @@ void BackwardProblem::handleEventB(const int iroot) {
     model->updateHeavisideB(&rootidx[iroot * model->ne]);
 }
 
-/**
- * handleDataPoint executes everything necessary for the handling of data
- * points for the backward problems
- *
- * @param it index of data point @type int
- */
+
```

Comment:
I assume these docs all went to the h files?

Reply:
Assumption correct.